neonvm/tools/vm-builder-generic/main.go

```diff
@@ -298,98 +308,40 @@ func main() {
 		log.Fatalln(err)
 	}
 
-	var SrcImageSpec ImageSpec
+	tmplCtx := TemplatesContext{
```

Comment:
s/Ctx/Args/ or s/Ctx/Opts/ (just because context.Context)

Reply:
Makes sense. Changed to `tmplArgs`.